src/components/Log/LogMessageAccordion.tsx

```diff
@@ -0,0 +1,70 @@
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Accordion, AccordionDetails, AccordionSummary, Box, BoxProps, Typography } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { MessageItem } from '../../pages/messages/components/MessageList';
+import { LogEntryType } from '../../types/log.types';
+import { Message } from '../../types/publication_types/ticket.types';
+import { dataTestId } from '../../utils/dataTestIds';
+import { logBackgroundColor } from './LogEntry';
+
+interface LogMessageAccordionProps {
+  messages: Message[];
+  type: LogEntryType;
```

Comment:
Tror bristen her kan være at vi sender inn `type` her, men den brukes bare for å bestemme farge. Så kanskje er det enklere å bare sende fargen direkte som prop her i stedet?

```suggestion
  bgcolor: BoxProps['bgcolor'];
```

Reply:
Ser nå at type setter både bakgrunn til hele log entry og meldinger, så da gir det mening å sende type ja 👍